install_script.sh.template

```diff
@@ -498,53 +498,49 @@ END
     fi
 }
 
-function configure_installer() {
-    local sudo_cmd="$1"
-    local trace_id="$2"
-    local apm_instrumentation_enabled="$3"
-
-    $sudo_cmd mkdir -p /etc/datadog-agent
-    $sudo_cmd bash -c "cat <<EOT > /etc/datadog-agent/datadog-installer.ini
-[features]
-apm_instrumentation=$apm_instrumentation_enabled
-
-[telemetry]
-trace_id=$trace_id
-parent_id=$trace_id
-priority=2
-EOT"
-}
-
 function _install_installer() {
     local sudo_cmd="$1"
     local os="$2"
     local distribution="$3"
-    local apm_instrumentation_enabled="$4"
+    local datadog_installer="$4"
+    local apm_instrumentation_enabled="$5"
```

Comment:
missed the cleanups of datadog_installer and apm_instrumentation_enabled which are now unused: check moved above